app/components/classification-select.hbs

```diff
@@ -6,7 +6,7 @@
     @searchMessage="Typ om te zoeken"
     @searchField="label"
     @options={{this.classifications.value}}
-    @selected={{this.selectedClassification}}
+    @selected={{@selected}}
```

Comment:
Indeed now that the component isn't used in a mu-search context that change is nice, can you remove the associated getter `get selectedClassification()` from `classification-select.js` as well, as we don't use it anymore ?

Reply:
No technically not there only one problem i see with how i did this and that when you reload the page. The field gets cleared while the filter stays active the rest just keeps their value but because I set the value in the index.js and not let it be handled by the component this one does lose it's value.

Reply:
Ah indeed but that is a problem in the multi select component, not in the default select, right ? But good catch, that needs to be fixed before merging